onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -222,11 +218,11 @@ static Status AddBinaryOperator(int32_t op_type,
 static Status AddSqueezeOp(ModelBuilder& model_builder,
                            const std::string& node_name,
                            const std::string& input, const std::string& output,
-                           vector<int32_t> axes) ORT_MUST_USE_RESULT;
+                           std::vector<int32_t> axes) ORT_MUST_USE_RESULT;
```

Comment:
general question, why do we have the function declaration right before the definition?

Reply:
This is to use the ORT_MUST_USE_RESULT, seems I cannot add `ORT_MUST_USE_RESULT` without separate decl and impl

Reply:
oh, now Status is `[[nodiscard]]` so we don't need `ORT_MUST_USE_RESULT` if that's the return type. could make this file a bit shorter :)
don't need to do it now though